core/dive.c

```diff
@@ -3429,18 +3429,6 @@ struct tag_entry *taglist_added(struct tag_entry *original_list, struct tag_entr
 	return added_list;
 }
 
-void dump_taglist(const char *intro, struct tag_entry *tl)
```

Comment:
I am not sure if this will be needed once somebody wants to debug something around tags.

Reply:
If somebody wants to debug, they could simply temporarily add two lines of the type
```
printf("Tags of dive %d\n", d->number);
for(struct tag_entry *tag = d->tag_list; tag; tag = tag->next) printf("%s (%s)\n", tag->tag->name, tag->tag->source);
```
at strategic positions, no?

I'm not convinced we want to drag around that function for the few cases when it might be needed. At least it should be ifdef-ed out.